fdbrpc/AsyncFileWriteChecker.cpp

```diff
@@ -32,10 +32,10 @@ static void compareWriteInfo(AsyncFileWriteChecker::WriteInfo w1, AsyncFileWrite
 class LRU2 {
 private:
 	struct node {
-		uint32_t page;
+		int page;
```

Comment:
Note that `int` is platform-dependent, the only guarantees for `int` are: 1. `sizeof(int) >= sizeof(short int) >= sizeof(char)`; 2. `sizeof(int)>=16`. I would recommend replacing `uint32_t` by `int32_t` instead.

See, e.g. C++ standard 6.8.2 [basic.types] for more details.